sklearn/feature_selection/base.py

```diff
@@ -120,3 +120,67 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def feature_wise_scorer(score_func):
+    """ A wrapper function around score functions.
+
+    Parameters
+    ----------
+    score_func : callable
+        Function taking array(s) X and/or y, and returning a pair of arrays
+        (scores, pvalues) or a single array with scores.
+
+    Returns
+    -------
+    scores : array, shape = [n_features,]
```

Comment:
You should document in the format

```
scores : array-like, shape (n_features,)
```

(At least that's the format in `sklearn/linear_model/logistic.py`. I know the documentation for these things to be inconsistent at times.)